x-pack/plugins/cloud_security_posture/public/common/hooks/use_navigate_findings.ts

```diff
@@ -35,21 +35,31 @@ const getFindingsQuery = (queryValue: Query['query']): Pick<FindingsBaseURLQuery
 export const useNavigateFindings = () => {
   const history = useHistory();
 
-  return (query?: Query['query']) => {
+  return (query: Query['query'] = {}) => {
     history.push({
       pathname: findingsNavigation.findings_default.path,
-      ...(query && { search: encodeQuery(getFindingsQuery(query)) }),
+      ...(query && {
+        search: encodeQuery({
+          ...getFindingsQuery(query),
+          filters: [],
+        }),
+      }),
     });
   };
 };
 
 export const useNavigateFindingsByResource = () => {
   const history = useHistory();
 
-  return (query?: Query['query']) => {
+  return (query: Query['query'] = {}) => {
     history.push({
       pathname: findingsNavigation.findings_by_resource.path,
-      ...(query && { search: encodeQuery(getFindingsQuery(query)) }),
+      ...(query && {
+        search: encodeQuery({
+          ...getFindingsQuery(query),
+          filters: [],
```

Comment:
this bit makes sure navigation happens with no filters, effectively removing any persisted app filters. 

since we no longer have an app, we should remove app filters completely and only sync with global filters. tracked in #144807